lib/src/subscription.dart

```diff
@@ -182,7 +182,7 @@ abstract class SubscriptionSet with IterableMixin<Subscription> implements Final
   Subscription operator [](int index) => elementAt(index);
 
   @override
-  _SubscriptionIterator get iterator => _SubscriptionIterator._(this);
+  Iterator<Subscription> get iterator => _SubscriptionIterator._(this);
```

Comment:
wow, how did we not notice this before

Reply:
It worked just fine before. It is just not good style to expose private classes on public methods.